src/Nest/_Generated/_Requests.generated.cs

```diff
@@ -2701,12 +2703,8 @@ public partial class ForceMergeRequest : PlainRequestBase<ForceMergeRequestParam
 		public long? MaxNumSegments { get => Q<long?>("max_num_segments"); set => Q("max_num_segments", value); }
 		///<summary>Specify whether the operation should only expunge deleted documents</summary>
 		public bool? OnlyExpungeDeletes { get => Q<bool?>("only_expunge_deletes"); set => Q("only_expunge_deletes", value); }
-		///<summary>TODO: ?</summary>
-		public string OperationThreading { get => Q<string>("operation_threading"); set => Q("operation_threading", value); }
-		///<summary>Specify whether the request should block until the merge process is finished (default: true)</summary>
-		public bool? WaitForMerge { get => Q<bool?>("wait_for_merge"); set => Q("wait_for_merge", value); }
```

Comment:
Needs to be patched back in, in order to not break binary compatibility

Reply:
:+1: